rebar.config

```diff
@@ -20,17 +20,17 @@
 
 {ex_doc, [
     {extras, [
-          {'CHANGELOG.md', #{title => <<"Changelog">>}},
-          {'README.md', #{title => <<"Overview">>}},
-          {'LICENSE.md', #{title => <<"License">>}}
+          {"CHANGELOG.md", [{title, ~B"Changelog"}]},
+          {~B"README.md", [{title, ~B"Overview"}]},
+          {~B"LICENSE.md", [{title, ~B"License"}]}
     ]},
-    {main, <<"readme">>},
-    {homepage_url, <<"https://github.com/starbelly/rebar3_ex_doc">>},
-    {source_url, <<"https://github.com/starbelly/rebar3_ex_doc">>},
-    {assets, <<"assets">>},
+    {main, ~B"readme"},
+    {homepage_url, ~B"https://github.com/starbelly/rebar3_ex_doc"},
+    {source_url, ~B"https://github.com/starbelly/rebar3_ex_doc"},
+    {assets, #{~B"assets" => ~B"assets"}},
```

Comment:
This seems odd, even if correct.

Reply:
I mean having an option `assets` that is a map from `assets` to `assets`. Where's upstream that I could compare (since the oddity might already be coming from there, if I'm understanding you correctly)? 😄